knowledge-base/how-to-remove-trial-message.md

```diff
@@ -0,0 +1,27 @@
+---
+title: How to remove trial message
+description: This article explains how to remove the trial message after the license is purchased
+type: how-to
+page_title: How to Remove the Trial Message After the License is Purchased
+slug: 
+position: 
+tags: how-to-remove-trial-message
+ticketid: 1459034
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Solution
+After the license is purchased you need to download the Telerik Reporting DEV installer from your [Telerik account -> Downloads](https://www.telerik.com/account/my-downloads). 
```

Comment:
I think that first, you need to uninstall the Trial version

Reply:
consider the link 'https://www.telerik.com/account/product-download?product=REPORTING' that will lead the reader directly to the Reporting product downloads